apps/web-giddh/src/app/inventory/components/welcome-inventory/welcome-inventory.scss

```diff
@@ -0,0 +1,8 @@
+@media (max-width:991px){
+    .col-xs-offset-4 {
```

Comment:
for sure... make BS class with nested CSS, if you added like this no any chances to distort another same name classes :)

Reply:
this html file contains single class name of this, so no chance of distorting other css